### DISCUSSION TITLE
lesson 2: license identifier error

### DISCUSSION THREAD

**Discussion by dodeyeikoi** [Cyfrin/foundry-full-course-cu#688] [Q&A]
I get this error when I use // SPDX-Licence-Identifier: MIT according to the lesson:

Warning: SPDX license identifier not provided in source file. Before publishing, consider adding a comment containing "SPDX-License-Identifier: <SPDX-License>" to each source file. Use "SPDX-License-Identifier: UNLICENSED" for non-open-source code. Please see https://spdx.org for more information.
--> SimpleStorage.sol

how do I solve this please?

**Comment by theirrationalone** [comment]
There is a typo in `Licence` and the typo is `c` last one that should be `s` ----> `License` is correct

`Licence` ❌
`License` ✅

**Reply by dodeyeikoi** [reply]
thanks! It worked

**Reply by theirrationalone** [reply]
You're Welcome @dodeyeikoi ! ❤

**Comment by CoggleChucker** [comment]
There is a typo in your line:
you typed "Licence".
correct spelling is "License".

let me know if the error warning still persists after the change.

**Reply by dodeyeikoi** [reply]
It worked, thank you!

**Comment by Mahfuz-13** [comment]
There is a typo error in your code 

```solidity
        // SPDX-"""Licence"""-Identifier: MIT 
        
        ```
        
        You should use License instead of Lincence
        
        
        The rest of your code is stunning Good Luck!!!
